### PR TITLE
[WasmGC article] Add Flutter demo

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Flutter demo.

### CODE REVIEW

File: site/en/blog/wasmgc/index.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ Now you may be wondering what the point is, since the Kotlin code above basicall
 In this early experimental stage, the [Kotlin/Wasm image viewer](https://kotlin-wasm-image-viewer.glitch.me/) demo is fully rendered onto a `canvas` with no meaningful DOM tree that could be used to create an [accessibility tree](https://developer.mozilla.org/docs/Glossary/Accessibility_tree). This means it's completely inaccessible to non-sighted users and breaks important browser features like find on page, translation, text selection, extensions, zooming, and link to text fragment.
 {% endAside %}
 
+The Dart and Flutter teams at Google are also preparing support for WasmGC support. The Dart-to-Wasm compilation work is almost complete and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). A demo showcasing the Flutter WasmGC Preview is embedded below.
```
Comment (tomayac) [inline site/en/blog/wasmgc/index.md:121]:
```suggestion
The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). A demo showcasing the Flutter WasmGC Preview is embedded below.
```

**Comment by tomayac** [comment]
@rachelandrew or whoever ends up reviewing this: please merge it from your end immediately, as tomorrow’s a public holiday here. Thank you !

File: site/en/blog/wasmgc/index.md [hunk 2]
```diff
@@ -89,6 +89,8 @@ To verify the real-world impact of this improvement, Chrome's Wasm team has comp
 
 ## Seeing a WasmGC-ported programming language in action
 
+### Kotlin/Wasm
```
Comment (rachelandrew) [inline site/en/blog/wasmgc/index.md:92]:
```suggestion
### Kotlin and Wasm
```

File: site/en/blog/wasmgc/index.md [hunk 3]
```diff
@@ -118,6 +120,12 @@ Now you may be wondering what the point is, since the Kotlin code above basicall
 In this early experimental stage, the [Kotlin/Wasm image viewer](https://kotlin-wasm-image-viewer.glitch.me/) demo is fully rendered onto a `canvas` with no meaningful DOM tree that could be used to create an [accessibility tree](https://developer.mozilla.org/docs/Glossary/Accessibility_tree). This means it's completely inaccessible to non-sighted users and breaks important browser features like find on page, translation, text selection, extensions, zooming, and link to text fragment.
 {% endAside %}
 
+### Dart and Flutter
+
+The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). The [Flutter WasmGC Preview](https://flutterweb-wasm.web.app/) demo is embedded below.
```
Comment (rachelandrew) [inline site/en/blog/wasmgc/index.md:125]:
```suggestion
The Dart and Flutter teams at Google are also preparing support for WasmGC. The Dart-to-Wasm compilation work is almost complete, and the team is working on tooling support for delivering Flutter web applications compiled to WebAssembly. You can read about the current state of the work in the [Flutter documentation](https://flutter.dev/wasm). The following demo is the [Flutter WasmGC Preview](https://flutterweb-wasm.web.app/).
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
See changes inline.

**Comment by tomayac** [comment]
It's actually officially Kotlin/,Wasm, with slash.

**Comment by rachelandrew** [comment]
> It's actually officially Kotlin/,Wasm, with slash.

You appeared to have just ignored the suggestion, rather than comment on it. Our style guide suggests not using the slash like that so unless I'm told specifically I'm going to change it.

**Comment by mraleph** [comment]
@rachelandrew `Kotlin/Wasm` refers to [this](https://kotlinlang.org/docs/wasm-overview.html). It the official name of the compilation target. They also have `Kotlin/Native` and `Kotlin/JS`.

**Comment by rachelandrew** [comment]
> @rachelandrew `Kotlin/Wasm` refers to [this](https://kotlinlang.org/docs/wasm-overview.html). It is the official name of the compilation target. They also have `Kotlin/Native` and `Kotlin/JS`.

yes that's not a problem, I was just explaining why I merged the suggestion as Thomas didn't respond to it to explain, and I was trying not to block.

**Comment by rachelandrew** [comment]
I also note that the page you sent me to has a title "Kotlin Wasm" and not "Kotlin/Wasm", the former would definitely be less confusing as what it looks like to anyone who doesn't know is "Kotlin and Wasm" or "Kotlin or Wasm". So I would suggest we use what the project is using on their site?

File: site/en/blog/wasmgc/index.md [hunk 4]
```diff
@@ -89,6 +89,8 @@ To verify the real-world impact of this improvement, Chrome's Wasm team has comp
 
 ## Seeing a WasmGC-ported programming language in action
 
+### Kotlin and Wasm
```
Comment (rachelandrew) [inline site/en/blog/wasmgc/index.md:92]:
```suggestion
### Kotlin Wasm
```

Comment (rachelandrew) [reply site/en/blog/wasmgc/index.md:92]: Changed to match https://kotlinlang.org/docs/wasm-overview.html

**Comment by mraleph** [comment]
> I also note that the page you sent me to has a title "Kotlin Wasm" and not "Kotlin/Wasm", the former would definitely be less confusing as what it looks like to anyone who doesn't know is "Kotlin and Wasm" or "Kotlin or Wasm". So I would suggest we use what the project is using on their site?

FWIW it has 12 unique occurrences of "Kotlin/Wasm" and 3 unique occurrences of "Kotlin Wasm". 

I am fairly sure most of their own documentation refers to it as Kotlin/Wasm.

**Comment by rachelandrew** [comment]
> > I also note that the page you sent me to has a title "Kotlin Wasm" and not "Kotlin/Wasm", the former would definitely be less confusing as what it looks like to anyone who doesn't know is "Kotlin and Wasm" or "Kotlin or Wasm". So I would suggest we use what the project is using on their site?
> 
> FWIW it has 12 unique occurrences of "Kotlin/Wasm" and 3 unique occurrences of "Kotlin Wasm".
> 
> I am fairly sure most of their own documentation refers to it as Kotlin/Wasm.

The page referenced uses the space, which is clearer for our audience, and given there is already use of that, that's what we'll use. I've added it as a suggestion inline.

**Comment by mraleph** [comment]
@rachelandrew it only uses "Kotlin Wasm" in the title and 2 other places. It uses Kotlin/Wasm in many more places. 
Screenshot 2023-11-01 at 14 25 00 (cc @bashorov).

Though to be honest it is not the hill I would die on. It's not even what this PR is about.

**Comment by mit-mit** [comment]
Yeah, let's get this PR merged, and then we can clean up the Kotlin/Wasm stuff (if deemed needed) in a second PR.

**Comment by mit-mit** [comment]
@rachelandrew can we please merge this? I believe Tom is OOO.

**Comment by rachelandrew** [comment]
Happy to merge it with the space, as I feel it's less confusing. If the docs are changed so they never use it with the space, fine. But to anyone who doesn't know the project (including me, as it was my confusion that started this) the slash is confusing, so if there's an option to avoid it, I'm going to use it.

**Comment by mraleph** [comment]
I am fine with the space. We can address this later.

**Comment by tomayac** [comment]
@rachelandrew We seem to have it in a mergeable state now… PTAL. Thanks!

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**